html/includes/print-nac.inc.php

```diff
@@ -0,0 +1,69 @@
+<?php
+echo '<table class="table table-hover table-condensed">
```

Comment:
You'll need to convert this table to use bootgrid.

We have lots using it already for examples.

Reply:
<img width="1438" alt="screen shot 2018-11-02 at 00 36 49" src="https://user-images.githubusercontent.com/15846428/47892648-9f2e9d80-de37-11e8-8147-7cc6d9caf34c.png">

Reply:
done. Thanks for advising.